docker.go

```diff
@@ -0,0 +1,44 @@
+package docker
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/client"
+	"github.com/loadimpact/k6/js/modules"
+)
+
+const version = "v0.0.2"
+
+func init() {
+	modules.Register("k6/x/docker", &Docker{
+		Version: version,
+	})
+}
+
+// Docker is the main export of k6 docker extension
+type Docker struct {
+	Version string
+}
+
+// Ps lists Docker containers
+func (d *Docker) Ps() []string {
+	cli, err := client.NewClientWithOpts(client.FromEnv)
+	if err != nil {
+		panic(err)
```

Comment:
I think it's better to return the error. But probably @simskij know better :)

Reply:
Yes, I agree that we should return the error here rather than panicking. The docker client erroring should probably not be considered fatal for the whole VU goroutine. 🤔

Reply:
✅